assignment/tool/src/webapp/vm/assignment/chef_assignments_student_review_edit.vm

```diff
@@ -5,6 +5,16 @@
 	focus_path = [ '$fField' ];
 	$(document).ready(function(){
 		ASN.setupPeerReviewAttachment();
+		
+        if($('.sakai-rubric-criteria-grading').length) {
+  			$('body').on('total-points-updated', function (e) {
+				e.stopPropagation();
+				var gradeField = $('#grade');
+				if (gradeField.length) {
+					gradeField.val(e.detail.value);
+				}
+  			});
+       	}
```

Comment:
You can do this without jQuery and I really want people to try and avoid jQuery unless it really helps keep code concise. In this case jQuery doesn't offer any advantages.

document.body.addEventListener("total-points-updated", (e) => {

   e.stopPropagation();
   const gradeField = document.getElementById("grade");
   gradeField && gradeField.value = e.detail.value;
});

I wouldn't bother with checking if the element is there. The function is never executed unless the event comes in.

Reply:
Thank you, Adrian, for your suggestion.
I've made the change and tested it.
@adrianfish